lib/flight_job/command.rb

```diff
@@ -25,72 +25,118 @@
 # https://github.com/openflighthpc/flight-job
 #==============================================================================
 
-require_relative 'template'
-require_relative 'list_output'
+require 'ostruct'
+require 'pastel'
 
 module FlightJob
   class Command
     attr_accessor :args, :opts
 
-    def initialize(*args, **opts)
+    def initialize(args, opts)
       @args = args.freeze
-      @opts = Hashie::Mash.new(opts)
+      @opts = opts
     end
 
     def run!
-      Config::CACHE.logger.info "Running: #{self.class}"
+      FlightJob.logger.info "Running: #{self.class}"
       run
-      Config::CACHE.logger.info 'Exited: 0'
+      FlightJob.logger.info 'Exited: 0'
     rescue => e
       if e.respond_to? :exit_code
-        Config::CACHE.logger.fatal "Exited: #{e.exit_code}"
+        FlightJob.logger.fatal "Exited: #{e.exit_code}"
       else
-        Config::CACHE.logger.fatal 'Exited non-zero'
+        FlightJob.logger.fatal 'Exited non-zero'
       end
-      Config::CACHE.logger.debug e.backtrace.reverse.join("\n")
-      Config::CACHE.logger.error "(#{e.class}) #{e.message}"
+      FlightJob.logger.debug e.backtrace.reverse.join("\n")
+      FlightJob.logger.error "(#{e.class}) #{e.message}"
       raise e
     end
 
     def run
       raise NotImplementedError
     end
 
-    def list_output
-      @list_output ||= ListOutput.build_output(verbose: opts.verbose)
+    def pastel
+      @pastel ||= Pastel.new
+    end
+
+    def output_options
+      {
+        verbose: (opts.verbose ? true : nil),
+        ascii: (opts.ascii ? true : nil),
+        interactive: (opts.ascii || opts.pretty || $stdout.tty? ? true : nil),
+        json: (opts.json ? true : nil)
+      }
     end
 
     def load_template(name_or_id)
+      template = Template.new(id: name_or_id)
+      if template.exists?
+        raise InternalError, <<~ERROR.chomp unless template.valid?(:verbose)
+          Can not load the following template as it is invalid: #{template.id}
```

Comment:
s/Can not/Cannot/

Reply:
95efe852ac5976e47000b00ede3f254b0b92a091